about.html

```diff
@@ -0,0 +1,129 @@
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li><a class="campaign" href="">CC Campaign</a></li>
+                </ul>
+            </nav>
+        </div>
+
+
+    </header>
+
+    <dialog class="modal_menu">
+        <div class="close">&times;</div>
+        <nav>
+            <ul>
+                <li><a href="about.html">About</a></li>
+                <li><a href="">Program</a></li>
+                <li><a href="">Join</a></li>
+                <li><a href="">Sponsor</a></li>
+                <li><a href="">News</a></li>
+                <li><a class="campaign" href="">CC Campaign</a></li>
+            </ul>
+        </nav>
+    </dialog>
+    <main class="container">
+        <section class="s_headline">
+            <div class="wrap">
+                <div class="wrapper1">
+                    <h1>DEFCON HACKER CONFERENCE 2023</h1>
+                    <hr class="line">
+                    <div class="description">
+                        <p class="des_mob">DEF CON started as talks and parties that evolved into a community of those who with a hacker’s mindset and an insatiable curiosity of how things work. We are committed to creating a place for all hackers to share, interact and learn from each other.</p>
+                        <p class="des_desk">DEF CON started as talks and parties that evolved into a community of those who with a hacker’s mindset and an insatiable curiosity of how things work. We are committed to creating a place for all hackers to share, interact and learn from each other. Sometimes, though, we have to reassess whether the structure of DEF CON is helping or hindering this vision.
+
+One example of this is how we categorize creators. There are a lot of content departments at DEF CON. If you’re a Vendor, you go here. If you’re a Village, you go over there. If something doesn’t quite fit our categories we force them to fit: The EFF or a University in our vendor room for example.</p>
+                    </div>
+                    <div class="address">
+                        <p>Please contact us per Email for further questions about DEFCON Conference 2023</p>
+                        <p class="email">defcon2023@defcon.org <span>&gt;</span></p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="s_logo">
+                <div class="wrapper_logo">
+                    <h2>DEFCON HACKER<br>CONFERENCE 2023 Logo</h2>
+                    <hr class="line">   
+                    <p>The logo of DEFCON hacker conference 2023 was decided through the logo competition from 8, June to 7, July.</p>
+                    <img src="static/images/defcon_logo.png" alt="CC global summit logo">
+                </div>
+            </div>            
+        </section>
```

Comment:
- [ ] Please complete the About page with the logo section. It contains a middle section with a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo. 👇🏽 

| Original design | Youe design |
|---|---|
| ![image](https://github.com/wacoo/DEF-CON_Summit/assets/58642949/8a3477ef-2d1b-46dd-8d77-5f1d3986ebb2) | ![image](https://github.com/wacoo/DEF-CON_Summit/assets/58642949/33b36791-def0-4c2a-8a2a-6c3e1e8166de) |